client/src/components/Feed/FiltersList.js

```diff
@@ -0,0 +1,76 @@
+// Core
+import React, { useContext } from "react";
+import styled from "styled-components";
+
+// Antd 
+import { Button } from "antd-mobile";
+
+// Svg
+import { ReactComponent as CloseIcon } from "assets/icons/close-btn.svg";
+
+// Local
+import { FeedContext } from "pages/Feed.js";
+
+// Constants
+import { theme } from "constants/theme";
+import { ROYAL_BLUE, WHITE } from "constants/colors";
+
+const TagButton = styled(Button)`
+  align-items: center;
+  background-color: ${WHITE};
```

Comment:
Let's replace the `background-color` and `color` properties with `theme.button.primary`.